packages/runtimes/service-worker/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@parcel/runtime-service-worker",
+  "version": "2.0.0-rc.0",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/parcel"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/parcel-bundler/parcel.git"
+  },
+  "main": "./lib/ServiceWorkerRuntime.js",
+  "source": {
+    "./lib/ServiceWorkerRuntime.js": "./src/ServiceWorkerRuntime.js"
+  },
+  "browser": "browser.js",
```

Comment:
This will definitely break when building the REPL.

Reply:
The thing is, runtimes don't override a dependency, they add an _additional_ resolution - the original is still there. For replacing async dependencies this works because we prefer resolutions in the same bundle, but for sync dependencies you'll get the original asset.

Reply:
I guess one option would be to create another package, eg @parcel/service-worker... but I was trying avoid that

Reply:
I guess that wouldn't be too bad. You'd have to install the package manually into your project anyway (e.g. in Yarn 2) even if the runtime is in the default config 🤷